frontend/i18next.scanner/fr/translation.json

```
@@ -348,5 +348,22 @@
   "Change avatar": "Changer l'avatar",
   "Maximum size: {{size}} MB": "Taille maximale: {{size}} Mo",
   "Cover image of {{publicName}}": "Image de couverture de {{publicName}}",
-  "Change cover": "Changer la couverture"
+  "Change cover": "Changer la couverture", 
+  "Comment_sub": "Commentaire", 
+  "New space": "Nouvel espace", 
+  "Space updated": "Espace mis à jour",
+  "Space deleted": "Espace supprimé",
+  "Space restored": "Espace restauré",
+  "Status updated": "Changement de statut", 
```

to have the same wording as `"Space updated": "Espace mis à jour"` maybe 
```suggestion
  "Status updated": "Statut mis à jour", 
```
(if you decide to apply, do the same fir content)